plugins/importexport/native/templates/resultsImport.tpl

```diff
@@ -0,0 +1,71 @@
+{**
+ * plugins/importexport/native/templates/results.tpl
+ *
+ * Copyright (c) 2014-2020 Simon Fraser University
+ * Copyright (c) 2003-2020 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * Result of operations this plugin performed
+ *}
+{if $errorsFound}
+	{translate key="plugins.importexport.native.processFailed"}
+{else}
+	{translate key="plugins.importexport.native.importComplete"}
+	<ul>
+		{foreach from=$importedRootObjects item=contentItemArrays key=contentItemName}
+			<b>{$contentItemName}</b>
+			{foreach from=$contentItemArrays item=contentItemArray}
+				{foreach from=$contentItemArray item=contentItem}
+					<li>
+						{$contentItem->getUIDisplayString()}
+					</li>
+				{/foreach}
+			{/foreach}
+		{/foreach}
+	</ul>
+{/if}
+
+{if array_key_exists('warnings', $errorsAndWarnings) && $errorsAndWarnings.warnings|@count > 0}
```

Comment:
(as elsewhere)

Reply:
Moved everything to pkp-lib and added a common `innerResults.tpl` template for the specific block